nbs/01_nbio.ipynb

```diff
@@ -446,6 +446,31 @@
     "finally: tmp.unlink()"
```

Comment:
<p> </p><blockquote>This returns an empty new notebook.</blockquote><p>You might want to put this in a docstring, and also have parameters for nbformat with defaults?  </p><p> </p><p>Also can you expand upon the intended use of the new notebook, if you have something in mind, it would be useful to show that here as an example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/execnb/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>It's more of a completeness thing: I'm dealing with git diffs between notebooks. The creation of a new file is represented as <code>(/dev/null, Foo.ipynb)</code> (where <code>/dev/null</code> indicates the non-existence state). I'm using <code>nbdime</code> to compare notebooks and its API expects two complete notebooks to compare so instead of passing <code>/dev/null</code> or <code>None</code> you pass an empty new notebook. I'm using <code>execnb</code> to read but if there's nothing to read I want a convenience method for creating an empty one.</p><p> </p><p>This use case might be a bit excessive for an explanation... it's more of a "CRUD" thing - the current API supports Read, Update (via write), I guess delete is just unlink... but no Create which would be accomplished via <code>write_nb(new_nb())</code></p><p> </p><p>As for <code>nbformat</code> parameters are you talking about: https://github.com/jupyter/nbformat/blob/main/nbformat/v4/nbbase.py#L163 ?        </p>